about.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title aria-label="title">Graphic Designer's Summit Website</title>
+  </head>
+  <body>
+    <!-- Navigation Section  -->
+
+    <header>
+      <nav class="top-header">
+        <ul class="top-nav">
+          <li class="top-item">
+            <a href="#" class="nav-link"
+              ><img src="img/facebook-icon.png" alt=""
+            /></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link"><img src="img/t-icon.png" alt=""/></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">English</a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">login</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="navbar">
+        <a href="index.html" class="nav-branding"
+          ><img src="img/logo1.png" alt="logo image"
+        /></a>
+        <ul class="nav-menu nav-menu1">
+          <li class="nav-item home-href">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">News</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">
+              <span> GD Campaign</span></a
+            >
+          </li>
+        </ul>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <section class="first-container" aria-label="who I am">
+      <section class="introduction about" aria-label="who I am">
+        <h3 class="hello-share">"Hello! Sharing world"</h3>
+        <h1>GRAPHIC DESIGNERS GLOBAL SUMMIT 2023</h1>
+        <p class="short-intro">
```

Comment:
### Your Homepage Title :red_circle:
![image](https://user-images.githubusercontent.com/93039359/210934863-5dfd331f-6dca-418b-9b22-e05f557fc1e4.png)

### Design Homepage Title :green_circle:
![image](https://user-images.githubusercontent.com/93039359/210934945-dcbbb641-68e1-44eb-974a-aa05ee20f5cc.png)

- [ ] You are not using the correct font which is **COCOGOOSE**. Kindly use **COCOGOOSE** font for your title :+1:

Reply:
I have implemented this